### PR TITLE
[Python] Document-extensions pages batch - Counters, Attachments [Replace C# samples]

### PR DESCRIPTION
**Pull request by reebhub**
Related issues:
[RDoc-2905](https://issues.hibernatingrhinos.com/issue/RDoc-2905/Python-Document-extensions-Counters-Overview-Replace-C-samples) Counters > Overview
[RDoc-2906](https://issues.hibernatingrhinos.com/issue/RDoc-2906/Python-Document-extensions-Counters-Creating-and-modifying-counters-Replace-C-samples) Counters > Creating and modifying counters
[RDoc-2907](https://issues.hibernatingrhinos.com/issue/RDoc-2907/Python-Document-extensions-Counters-Delete-counter-Replace-C-samples) Counters > Delete counter
[RDoc-2908](https://issues.hibernatingrhinos.com/issue/RDoc-2908/Python-Document-extensions-Counters-Retrieve-counter-value-Replace-C-samples) Counters > Retrieve counter value
[RDoc-2909](https://issues.hibernatingrhinos.com/issue/RDoc-2909/Python-Document-extensions-Counters-Counters-and-other-features-Replace-C-samples) Counters > Counters and other features
[RDoc-2910](https://issues.hibernatingrhinos.com/issue/RDoc-2910/Python-Document-extensions-Counters-Indexing-counters-Replace-C-samples) Counters > Indexing counters
[RDoc-2911](https://issues.hibernatingrhinos.com/issue/RDoc-2911/Python-Document-extensions-Attachments-Storing-Replace-C-samples) Attachments > Storing
[RDoc-2912](https://issues.hibernatingrhinos.com/issue/RDoc-2912/Python-Document-extensions-Attachments-Loading-Replace-C-samples) Attachments > Loading
[RDoc-2913](https://issues.hibernatingrhinos.com/issue/RDoc-2913/Python-Document-extensions-Attachments-Deleting-Replace-C-samples) Attachments > Deleting
[RDoc-2914](https://issues.hibernatingrhinos.com/issue/RDoc-2914/Python-Document-extensions-Attachments-Copy-move-rename-Replace-C-samples) Attachments > Copy, move & rename
[RDoc-2915](https://issues.hibernatingrhinos.com/issue/RDoc-2915/Python-Document-extensions-Attachments-Indexing-Replace-C-samples) Attachments > Indexing
[RDoc-2916](https://issues.hibernatingrhinos.com/issue/RDoc-2916/Python-Document-extensions-Attachments-Bulk-insert-Replace-C-samples) Attachments > Bulk insert

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+{PANEL/}
+
+{PANEL: Syntax}
+
+{CODE:python Increment-definition@DocumentExtensions\Counters\Counters.py /}
+
+| Parameter     | Type    | Description      |
+|---------------|---------|------------------|
+| `counter_name` | string  | Counter's name  |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown:53]:
```suggestion
| `counter_name` | str | Counter's name  |
```

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+| `delta`       | long    | Increase Counter by this value.<br>Default value is 1.<br>For a new Counter, this number will be its initial value. |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown:54]:
```suggestion
| `delta`       | int    | Increase Counter by this value.<br>Default value is 1.<br>For a new Counter, this number will be its initial value. |
```

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+public class AbstractJavaScriptCountersIndexCreationTask : AbstractCountersIndexCreationTask
+{
+    public HashSet<string> Maps;
+    protected string Reduce;
+}
+{CODE-BLOCK/}
+
+| Property | Type | Description |
+| - | - | - |
+| **Maps** | `HashSet<string>` | The set of javascript map functions |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown:44]:
```suggestion
| **Maps** | `Set[str]` | The set of javascript map functions |
```

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+| **Reduce** | `string` | The javascript reduce function |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown:45]:
```suggestion
| **reduce** | `str` | The javascript reduce function |
```

**Review by poissoncorp — APPROVED** [review]

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown:53]: done

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/create-or-modify.python.markdown:54]: done

File: Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown (hunk 4 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/document-extensions/counters/indexing.python.markdown:45]: done

**Pull request merged**